Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -205,6 +205,7 @@ extension KeyboardShortcuts {
 				self.stringValue = "\(shortcut)"
 				self.showsCancelButton = true
 
+				userDefaultsRemove(name: self.shortcutName)
 				userDefaultsSet(name: self.shortcutName, shortcut: shortcut)
```

Comment:
I think it would be better for `userDefaultsSet` to handle unregistering any existing shortcut.

Reply:
I've updated my PR, by calling the `userDefaultsRemove` method inside `userDefaultsSet`. Is it the way you had in mind?